content/applications/inventory_and_mrp/manufacturing/plm/engineering_change_orders.rst

```diff
@@ -0,0 +1,289 @@
+
+View ECO
+--------
+
+To review the proposed changes, go to the `BOM Changes` ECO type from :menuselection:`PLM app -->
+Overview`, and click on the card to view the contained |ECOs|. Click on the newly created |ECO| in
+the `New` stage, and view a summary of the proposed changes in the :guilabel:`Operation Changes`
+tab. To implement the changes, click the :guilabel:`Revision` smart button.
+
+.. example::
+   An |ECO| adding another check for broken components is created in the `BOM Changes` ECO type
+   found in :menuselection:`PLM app --> Overview`. By default, |ECOs| created from tablet view are
+   named with the manufacturing order number for reference.
+
+   .. image:: engineering_change_orders/view-BOM-change.png
+      :align: center
+      :alt: Find the new ECO in the "BOM Changes" ECO type, in the *New* stage.
+
+On the new |BOM|, switch to the :guilabel:`Operations` tab, and select the :guilabel:`☰ (Show
+Instructions)` icon. Doing so opens a list of :guilabel:`Steps` to perform the operation, with the
+newest instruction titled `New Step Suggestion:`, followed by the user-entered title. Click the line
+item to view the suggested changes.
+
+.. image:: engineering_change_orders/show-instructions.png
+   :align: center
+   :alt: "Show Instructions" icon in the *Operations* tab of a BoM.
+
+On the :ref:`quality control point <quality/quality_control_points>` form, ensure the following form
+fields are accurately filled out to give detailed instructions for operators:
+
+- :guilabel:`Title`: rename to give a concise description of the new instruction
+- :guilabel:`Control per`: using the drop-down menu, determine whether this instruction applies
+  broadly for the :guilabel:`Product`, specifically for this :guilabel:`Operation` *only*, or a
+  particular :guilabel:`Quantity` of the product
+- :guilabel:`Type`: categorizes the control point type. From the drop-down menu, select
+  :guilabel:`Instructions` to detail an instruction for the worker. To receive input from the
+  workers, select the :guilabel:`Take a Picture`, :guilabel:`Register Consumed Materials`,
+  :guilabel:`Print Label`, or other :ref:`quality check options <quality/quality_control_points>`.
+
+Once the quality control point is configured, return to the :guilabel:`Steps` list using the
+breadcrumbs. Finally, drag the last quality control line item to its intended order of instructions.
+
+.. example::
+   Drag and reorder the `Check for broken switches` instruction, by clicking and dragging its "6
+   dots" icon to move it from the bottom to the second position.
+
+   .. image:: engineering_change_orders/reorder.png
+      :align: center
+      :alt: Drag and reorder instructions by selecting the "6 dots" icon on the far left.
+
+.. _plm/eco/eco-type:
+
+Create ECO type
+===============
+
+To access and manage ECO types, navigate to :menuselection:`PLM app --> Configuration --> ECO
+Types`.
+
```

Comment:
a screenshot of a blank ECO Type form could be beneficial here.

Reply:
To expand on that, try to fill in forms with relevant example data when possible to visually demonstrate *how* to fill in the form, and with *what* types of information 👍